folly/SocketAddress.cpp

```diff
@@ -125,6 +125,10 @@ struct GetAddrInfoError {
 
 namespace folly {
 
+#ifndef AI_NUMERICSERV		
+#define AI_NUMERICSERV 0		
```

Comment:
Where would this normally be defined? Most of the compatibility defines for sockets live in [folly/net/NetOps.h](https://github.com/facebook/folly/blob/main/folly/net/NetOps.h).

Reply:
I will check and move it accordingly, thank you.

Reply:
@Orvid Moved to the respective header as you recommended.